presto-hive/src/main/java/io/prestosql/plugin/hive/s3/S3SecurityMappingConfigurationProvider.java

```diff
@@ -90,7 +90,12 @@ public void updateConfiguration(Configuration configuration, HdfsContext context
 
         S3SecurityMapping mapping = mappings.get().getMapping(context.getIdentity(), uri)
                 .orElseThrow(() -> new AccessDeniedException("No matching S3 security mapping"));
+        if (mapping.isUseClusterDefault()) return;
```

Comment:
```suggestion
        if (mapping.isUseClusterDefault()) {
             return;
         }
```

Please always use curly brackets.